tests/test_operator.py

```diff
@@ -1243,25 +1221,30 @@ def test_022(self):
             "ErrImagePull"
         )
         with Then("CHI should be able to delete"):
-            kubectl.launch(f"delete chi {chi}", ok_to_fail=True, timeout=60)
+            kubectl.launch(f"delete chi {chi}", ok_to_fail=True, timeout=600)
             assert kubectl.get_count("chi", f"{chi}") == 0
 
+
 @TestScenario
 @Name("test_023. Test auto templates")
 def test_023(self):
+    chit_data = manifest.get_chit_data(util.get_full_path("templates/tpl-clickhouse-auto.yaml"))
+    expected_image=chit_data['spec']['templates']['podTemplates'][0]['spec']['containers'][0]['image']
     kubectl.create_and_check(
         config="configs/test-001.yaml",
         check={
             "pod_count": 1,
             "apply_templates": {
+                settings.clickhouse_template,
                 "templates/tpl-clickhouse-auto.yaml",
             },
             # test-001.yaml does not have a template reference but should get correct ClickHouse version
-            "pod_image": settings.clickhouse_version,
+            "pod_image": expected_image,
```

Comment:
this test case worked wrong, we had the same versions in `tpl-clickhouse-auto.yaml` and `tpl-clickhouse-stable.yaml`